sdp.go

```diff
@@ -93,6 +92,10 @@ func trackDetailsFromSDP(log logging.LeveledLogger, s *sdp.SessionDescription) m
 				}
 
 			case sdp.AttrKeySSRC:
+				if useRid {
```

Comment:
I wouldn't make this a warn, may confuse people who are using Simulcast.

I would warn if a section no SSRC or Rid though!